qiskit/tools/visualization/_circuit_visualization.py

```diff
@@ -441,4 +452,4 @@ def _matplotlib_circuit_drawer(circuit,
                                        plot_barriers=plot_barriers,
                                        reverse_bits=reverse_bits)
     qcd.parse_circuit(circuit)
-    return qcd.draw(filename)
+    return qcd.draw(filename, verbose=True)
```

Comment:
`verbose` prints in stdout. Is this change something to clean up?

Reply:
Yes, removed it now!